mautic/mautic
# Add not exist greater than/ less then for segment filters

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Added segment filter not exist greater than/less then.
As follow up of this PR https://github.com/mautic/mautic/pull/7122/ (most recently https://github.com/mautic/mautic/pull/8652)

This allow use filters for example:

- contacts with orders but not last_active from last month etc

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.  Create custom datetime 
3. If today is 14. 1. 2019 set this custom field to 10. 1. 2019
4. Got to segments with operator  `not exists greater than` 11 .1. 2019
5: Wait
6. Contact should be added to segment
7. Change the filter to not exists greater than 9. 1. 2019
8. Contact should be removed from segment
9. Repeat same for `not exist less than` operator
